support-services/src/test/scala/com/gu/support/catalog/CatalogServiceIntegrationSpec.scala

```diff
@@ -31,7 +31,7 @@ class CatalogServiceIntegrationSpec extends AsyncFlatSpec with Matchers with Ins
     forAll(product.ratePlans(environment))(
       ratePlan =>
         service.getPriceList(ratePlan).fold {
-          Console.println(s"Failed to find a catalog price list for $environment > $product > ${ratePlan.id}")
+          Console.println(s"Failed to find a catalog price list for $environment > $product > ${ratePlan.billingPeriod} > ${ratePlan.id}")
```

Comment:
Unrelated?

Reply:
It's related to the issue with a rogue lambda updating all the product rate plan ids on UAT which broke an integration test and stopped me pushing this branch. This made it easier to debug so I want to leave it in

Reply:
Ok, makes sense